libs/vertexai/langchain_google_vertexai/_anthropic_utils.py

```diff
@@ -47,16 +48,24 @@ def _format_image(image_url: str) -> Dict:
     """Formats a message image to a dict for anthropic api."""
     regex = r"^data:(?P<media_type>image/.+);base64,(?P<data>.+)$"
     match = re.match(regex, image_url)
-    if match is None:
+
+    if match:
+        return {
+            "type": "base64",
+            "media_type": match.group("media_type"),
+            "data": match.group("data"),
+        }
+    elif validators.url(image_url):
+        return {
+            "type": "url",
+            "url": image_url,
```

Comment:
ideally, we should support `gs://` urls too

afaik, we have a functionaly to fetch bytes content from GCS in utils, right?